rbreslow/zs
# Reloading a Hammer at doors deals high damage hit

## Overview

When equipped with the hammer, pressing reload when facing and in melee range of a `prop_door_rotating` the player will perform a 150dmg melee attack on the door. 

Resolves #23 

## Notes

Since this isn't an actual melee attack, it will only be performed if there is a valid target, otherwise reloading will do nothing

## Testing Instructions

* Join the server as a human on any map with at least 1 `prop_door_rotating` (for example.`zs_bdf_b3`)
* Obtain a Hammer (worth menu, purchase, etc)
* Find a door and press reload with the hammer equipped. When within range and looking at the door, you should melee attack the door for 150dmg.
* Pressing reload when out of range or not looking at a door will do nothing.

## Checklist
